modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -52,18 +48,21 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    <component>:
+    <component>: #<1>
       volumeClaimTemplate:
         spec:
-          storageClassName: <storage_class>
+          storageClassName: <storage_class> #<2>
           resources:
             requests:
-              storage: <amount_of_storage>
+              storage: <amount_of_storage> #<3>
 ----
+<1> Specify the core monitoring component for which you want to configure the PVC.
+<2> Specify an existing storage class.
```

Comment:
```suggestion
<2> Specify an existing storage class. it will use the default storage class if the value is not set
```
example
```
$ oc -n openshift-monitoring get ConfigMap cluster-monitoring-config -oyaml
apiVersion: v1
data:
  config.yaml: |
    prometheusK8s:
      volumeClaimTemplate:
        spec:
          storageClassName:
          resources:
            requests:
              storage: 10Gi
kind: ConfigMap
metadata:
  creationTimestamp: "2024-06-06T02:48:03Z"
  name: cluster-monitoring-config
  namespace: openshift-monitoring
  resourceVersion: "47934"
  uid: 46168aa2-e93e-4f12-b66e-9cce39d04556

$ oc get pv
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                                     STORAGECLASS   REASON   AGE
pvc-4ac0349b-7b12-46ce-aefb-ddb5f21d93c4   10Gi       RWO            Delete           Bound    openshift-monitoring/prometheus-k8s-db-prometheus-k8s-0   managed-csi             2m35s
pvc-68a4de46-1ffa-48de-a2cb-4bd5e4cfe104   10Gi       RWO            Delete           Bound    openshift-monitoring/prometheus-k8s-db-prometheus-k8s-1   managed-csi             2m35s
$ oc get sc
NAME                    PROVISIONER          RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
azurefile-csi           file.csi.azure.com   Delete          Immediate              true                   67m
managed-csi (default)   disk.csi.azure.com   Delete          WaitForFirstConsumer   true                   67m

```